libs/ledger-live-common/src/families/evm/api/etherscan.ts

```diff
@@ -18,11 +19,22 @@ async function fetchWithRetries<T>(
   retries = DEFAULT_RETRIES_API
 ): Promise<T> {
   try {
-    const { data } = await axios.request<{
+    const { data } = await network<{
       status: string;
       message: string;
       result: T;
-    }>(params);
+    }>({
+      ...params,
+      headers: {
+        ...(params?.headers || {}),
+        // We enforce a user-agent in order to prevent some restrictions that are being applied on not whitelisted user-agents
+        // E.g. moonscan is refusing `okhttp/4.9.2` (the android user-agent) to get the API response without going through the
+        // DDoS protection on cloudflare which leads to a 403 error
+        // This fix should be removed or improved ASAP.
```

Comment:
Any idea how? You and I both know this comment is here to stay :see_no_evil: 

Also just in case, can't these UA cause issues on other etherscan-like? I guess not if they're whitelisted, no need to non-reg other EVMs then?

Reply:
I'm not sure it's a feature that is here to stay on the Moonscan side tbh, so imo we need to discuss with them and remove that thing as soon as they add "okhttp" as a whitelisted user-agent.

So my guess is that I should be able to remove that pretty fast.

Regarding the other EVMs, I'll run the bot to make sure everything is fine 👌